tasks/install_windows.yml

```diff
@@ -0,0 +1,63 @@
+---
+# File: install_remote.yml - package installation tasks for Consul
+
+- name: Validate remote Consul directory
+  win_file:
+    path: /tmp/consul
+    state: directory
+
+- name: Verify TLS1.2 is used
+  win_regedit:
+    path: HKLM:\SOFTWARE\Microsoft\.NETFramework\v4.0.30319
+    name: SchUseStrongCrypto
+    data: 1
+    type: dword
+
+- name: Read Consul package checksum file
+  win_stat:
+    path: "/tmp/consul/consul_{{ consul_version }}_SHA256SUMS"
+  register: consul_checksum
+  tags: installation
+
+- name: Download Consul package checksum file
+  win_get_url:
+    url: "{{ consul_checksum_file_url }}"
+    dest: "/tmp/consul/consul_{{ consul_version }}_SHA256SUMS"
+  tags: installation
+  when: not consul_checksum.stat.exists | bool
+
+- name: Read Consul package checksum
+  win_shell: "findstr {{ consul_pkg }} /tmp/consul/consul_{{ consul_version }}_SHA256SUMS"
+  register: consul_pkg_checksum
+  tags: installation
+
+- name: Download Consul
+  win_get_url:
+    url: "{{ consul_zip_url }}"
+    dest: "/tmp/consul/{{ consul_pkg }}"
+  tags: installation
+
+- name: Calculate checksum
```

Comment:
These next two tasks are done different from the usual path, because `win_get_url` doesn't allow checksums like `get_url` does. Minor inconsistency we have to handle by hand here.